src/cmd/root.go

```diff
@@ -63,7 +63,7 @@ func init() {
 	viper.BindEnv("workers", "OPSLEVEL_WORKERS", "OL_WORKERS")
 	viper.BindEnv("disable-service-create", "OPSLEVEL_DISABLE_SERVICE_CREATE", "OL_DISABLE_SERVICE_CREATE")
 	cobra.OnInitialize(func() {
-		readConfig()
```

Comment:
1. Don't read config unnecessarily at start of program, read them when a command that needs input starts. 
2. Added `setupEnv()` because ENV can be relevant to a command even if config is not.